.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,194 @@
+          if-no-files-found: ignore
+
+      - name: Analyze E2E test results
+        id: analyze
+        if: always()
+        run: |
+          if [ -f "e2e-results.json" ]; then
+            # Parse JSON results
+            TOTAL_TESTS=$(jq '.stats.expected' e2e-results.json)
+            PASSED_TESTS=$(jq '.stats.passed' e2e-results.json)
+            FAILED_TESTS=$(jq '.stats.failed' e2e-results.json)
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)